### ISSUE TITLE
Allow multiple merge keys (syntax. like ::each and ::call)

### ISSUE THREAD

**Issue by scravy** [scravy/jinsi#11]
`::each` and `::call` have their arguments within the key, i.e.

```
::each >arguments here<: ...
```

and

```
::call template:
  arg1: ...
  arg2: ...
```

The same syntax could be used for `::merge` to allow multiple merge keys in an object, i.e. we can have:

```
::let:
  some:
    foo: one
    bar: two
  more:
    qux: three
    quuz: four
object:
  property: value
  ::merge some: {}
  ::merge more: {}
```

The actual argument should be ignored (could also be `null` instead of the empty object `{}`).

Depends on #10 . #3 would be nice to have for this too.
